credmgr

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+function credmgr_on() {
+	if ! [ -e "${HOME}/.aws/credentials.normal" ]; then
```

Comment:
In `setup_dev.sh`, you used `~/.aws/config`.  Did you mean to use that here?  Or is this just a result of the weirdness of Amazon's config scheme?